src/amo/utils/index.js

```diff
@@ -56,25 +56,30 @@ export const getCanonicalURL = ({
   return `${_config.get('baseURL')}${locationPathname}`;
 };
 
-export const isInternalURL = ({
+export const checkInternalURL = ({
   _config = config,
   urlString,
 }: {|
   _config?: typeof config,
   urlString: string,
-|}): boolean => {
+|}): { isInternalURL: boolean, strippedURL: string } => {
   const baseURL = _config.get('baseURL');
   const urlParts = url.parse(urlString, true);
+  const strippedURL = urlString.replace(baseURL, '');
```

Comment:
It's possible to configure a base URL so that `strippedURL` will not start with a slash. Since that would break things, it might be prudent to prefix this with a slash in case it's not.